brave-grpc/src/main/java/com/github/kristofa/brave/grpc/ClientRemoteEndpointExtractor.java

```diff
@@ -0,0 +1,9 @@
+package com.github.kristofa.brave.grpc;
+
+import com.twitter.zipkin.gen.Endpoint;
+import io.grpc.Channel;
+import io.grpc.MethodDescriptor;
+
+public interface ClientRemoteEndpointExtractor {
+    Endpoint remoteEndpoint(Channel channel, MethodDescriptor<?, ?> method);
```

Comment:
why would we do this as opposed to having people override the GrpcClientRequestAdapter?